src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13004,7 +12965,6 @@ Real64 VRFTerminalUnitEquipment::CalVRFTUAirFlowRate_FluidTCtrl(EnergyPlusData &
         using SingleDuct::SimATMixer;
 
         int constexpr Mode(1);  // Performance mode for MultiMode DX coil. Always 1 for other coil types
-        int OAMixNode;          // index to the mix node of OA mixer
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13007]:(style),[variableScope],The scope of the variable 'OAMixNode' can be reduced.